h2o-core/src/main/java/hex/grid/filter/StrictFilterFunction.java

```diff
@@ -0,0 +1,32 @@
+package hex.grid.filter;
+
+import hex.Model;
+
+import java.util.function.Function;
+
+public class StrictFilterFunction<MP extends Model.Parameters> implements PermutationFilterFunction<MP> {
```

Comment:
Probably not necessary if you add the defaults to the interface as mentioned above

Reply:
still relevant: probably not needed as `PermutationFilterFunction` == `FilterFunction` and this one is a `FunctionalInterface`, so you can achieve the same just with a lambda